### ISSUE TITLE
content_v2 API not being re-generated despite changes to discovery doc

### ISSUE THREAD

**Issue by sstrickl** [google/google-api-ruby-client#691]
Hi all,

There are recent changes in the content_v2 API (like new methods) that were released on 6/18 and have been reflected in the [discovery doc](https://www.googleapis.com/discovery/v1/apis/content/v2/rest). However, this API hasn't been regenerated since 6/12 and I'm unsure why. Please regenerate it as soon as possible, thanks! (If you know why it's not being regenerated, please let me know as well!)

**Comment by mattwhisenhunt** [comment]
My guess is the auto-generator won't replace older classes when it detects a conflict:
```
    conflict  google/apis/content_v2.rb
Overwrite /Users/googler/ruby/google-api-ruby-client/generated/google/apis/content_v2.rb? (enter "h" for help) [Ynaqdh] Y
       force  google/apis/content_v2.rb
    conflict  google/apis/content_v2/service.rb
Overwrite /Users/googler/ruby/google-api-ruby-client/generated/google/apis/content_v2/service.rb? (enter "h" for help) [Ynaqdh] 
       force  google/apis/content_v2/service.rb
    conflict  google/apis/content_v2/classes.rb
Overwrite /Users/googler/ruby/google-api-ruby-client/generated/google/apis/content_v2/classes.rb? (enter "h" for help) [Ynaqdh] 
       force  google/apis/content_v2/classes.rb
    conflict  google/apis/content_v2/representations.rb
Overwrite /Users/googler/ruby/google-api-ruby-client/generated/google/apis/content_v2/representations.rb? (enter "h" for help) [Ynaqdh] 
       force  google/apis/content_v2/representations.rb
```

**Comment by JustinBeckwith** [comment]
@dazuma there's an internal bug on this one as well

**Comment by dazuma** [comment]
@mattwhisenhunt I merged #692. However, it looks like there's now some urgency on this one, to get a gem released within the next few days. Are you familiar with how the gem auto release works? Is it possible to trigger it manually?

**Comment by dazuma** [comment]
Should be fixed in google-api-client version 0.23.2 (https://rubygems.org/gems/google-api-client/versions/0.23.2). This was apparently a symptom of a larger problem that we'll address in the toolchain (https://github.com/googleapis/discovery-artifact-manager/issues/86).
